config/default/core.entity_view_display.node.blog.default.yml

```diff
@@ -47,24 +40,16 @@ content:
     third_party_settings: {  }
     weight: 0
     region: content
-  field_social_sharing:
-    type: boolean
-    label: above
-    settings:
-      format: default
-      format_custom_false: ''
-      format_custom_true: ''
-    third_party_settings: {  }
-    weight: 3
-    region: content
 hidden:
   addtoany: true
   field_blog_type: true
   field_category: true
+  field_contact: true
```

Comment:
Could you please double-check whether we need these changes?

Reply:
reverted